api/test/unit/serializers/audio_serializers_test.py

```diff
@@ -0,0 +1,38 @@
+import uuid
+from unittest.mock import MagicMock
+
+from rest_framework.request import Request
+from rest_framework.test import APIRequestFactory
+
+import pytest
+
+from catalog.api.serializers.audio_serializers import AudioSerializer
```

Comment:
Why is this only tested against the AudioSerializer?

Could we write parameterised tests for this that automatically test this behaviour against all subclasses?

Reply:
That's a good idea. Implemented in 9e5a882.